feature/gribi/otg_tests/mpls_in_udp_scale/README.md

```diff
@@ -50,9 +50,24 @@ outer_ip-ttl =        "64"
   * Update policer-policies in a batch of `sched_q` = 1,000
   * Policer-policies changes should take effect within `sched_r` / 2 time
 
-#### Scale profile A - many vlans
+#### Scale profile A - many vlans 1 NH per NHG with same MPLS label
 
 * 20 ip destinations * 1,000 vlans = 20,000 'flows'
+* 1000 VRF have 1000 policy forwarding NHG with 8 NH
+* 1 default route in each VRF pointing to policy forwarding NHG.
+* Create 20K gRIBI NHG with 1 NH per NHG with same MPLS label.
+* 100K longest exact match route entries with 5 pointing to 1 gRIBI NHG.
```

Comment:
Clarify that these are gRIBI programmed prefixes.  (ie: they are not static routes, right)?  So it should read tsomething like:

Create 5 unique gRIBI prefixes which point to 1 NHG and repeat this 20k times, resulting in 100K total prefixes, pointing to 20k NHG.

Is that your intent?